### DISCUSSION TITLE
What happens when setting the bounding_box larger than the extracts bbox

### DISCUSSION THREAD

**Discussion by paulfi94** [systemed/tilemaker#523] [Q&A] [answered]
Hello,

i successfully generated vector tiles with tilemaker for europe. I also got it to generate the sea tiles around it.

Now the Problem is that the generated bbox for europe prevents the generation of those ocean tiles for the whole planet on low zoom levels. 
I found the `bounding_box` option. But i suspect setting it to cover the planet will result in a generation of many blank tiles on higher zoom levels and also in long generation times.

My questions are: 
1. Is it correct that setting the `bounding_box` to cover the planet will result in many blank tiles, long generation times and much memory usage?
2. What is the correct way to achieve a generation of oceans tiles for low zoom outside the automatically calculated bbox?

**Comment by systemed** [comment, marked as answer]
Probably the easiest way to do it right now is to generate the lowzoom ocean tiles in one run; then do another run with the `--merge` option for the Europe tiles. (Or vice versa!)
